waku/v2/node/wakunode2.nim

```diff
@@ -80,15 +80,16 @@ type
     rng*: ref BrHmacDrbgContext
     started*: bool # Indicates that node has started listening
 
-func protocolMatcher(codec: string): Matcher =
+proc protocolMatcher(codec: string): Matcher =
```

Comment:
Is this proc going to support more than one codec? If no, then I am wondering why not using simple string comparison?